src/player.html

```diff
@@ -184,7 +208,7 @@ <h1>{{gameState.splashtext}}</h1>
 			<section class="lightbox" ng-class="{ 'show': lightboxTarget != -1}" ng-hide="lightboxTarget == -1" ng-click="setLightboxTarget(-1)">
 				<div class="lightbox-content">
 					<button class="lightbox-close" ng-click="setLightboxTarget(-1)">X</button>
-					<iframe class="lightbox-video" ng-if="answers[lightboxTarget].options.asset.type == 'video'" ng-src="{{answers[lightboxTarget].options.asset.value}}" frameborder="0" allowfullscreen></iframe>
+					<iframe class="lightbox-iframe" ng-if="answers[lightboxTarget].options.asset.type == 'video' || answers[lightboxTarget].options.asset.type == 'image'" ng-src="{{answers[lightboxTarget].options.asset.value}}" frameborder="0" allowfullscreen></iframe>
```

Comment:
Like I mentioned in my review comments, this should probably be a `div` instead of an `iframe` for the image version.